Tests/FluentAssertions.Specs/Primitives/ObjectAssertionSpecs.cs

```diff
@@ -39,7 +39,8 @@ public void Should_throw_a_helpful_error_when_accidentally_using_equals()
 
     internal class DumbObjectEqualityComparer : IEqualityComparer<object>
     {
-        public new bool Equals(object x, object y) => x.Equals(y);
+        // ReSharper disable once MemberHidesStaticFromOuterClass
+        public new bool Equals(object x, object y) => x is not null && x.Equals(y);
```

Comment:
It we want to be correct, the two changed `Equals` methods in this file does not handle when both arguments are null.